### ISSUE TITLE
Implement and run metric based ad attacks against collection of classifiers

### ISSUE THREAD

**Issue by MartinLNG** [MartinLNG/adversarial_training_mps#161]
**Collection of classifiers:**
- Pretrained MPS
- GAN-trained MPS
- Adversarially trained MPS
- Ad. train + GAN train (Do this later, not part of the first minimal run)

**Performance metrics:**
- classification accuracy on validation set against strength of attack ($\delta$)

**Adversarial attacks:**
- Fast Gradient Sign Method
- Maybe Projection method aswell (especially for the toy data)
